### DISCUSSION TITLE
Downloading ZFS el7_9 gives 403 forbidden error

### DISCUSSION THREAD

**Discussion by saumitrasapre** [openzfs/zfs#12887] [Q&A] [answered]
My configuration is as follows - 
Distribution Name - Centos 7.9.2009
Kernel Version - 3.10.0-1160.31.1.el7.x86_64
Architecture - AMD64

I tried to download the ZFS rpm by executing the command - 
`rpm -Uvh http://download.zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm`

But this gave me the error - 
```
Retrieving http://download.zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm
curl: (22) The requested URL returned error: 403 Forbidden
error: skipping http://download.zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm - transfer failed
```

Similarly, attempting a wget also gave me a similar error - 
Command isssued - 

`wget http://download.zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm`

Error - 
```
--2021-12-20 00:25:55--  http://download.zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm
Resolving download.zfsonlinux.org (download.zfsonlinux.org)... 52.218.232.73, 2600:1fa0:408c:548:34da:c0a1::
Connecting to download.zfsonlinux.org (download.zfsonlinux.org)|52.218.232.73|:80... connected.
HTTP request sent, awaiting response... 403 Forbidden
2021-12-20 00:25:55 ERROR 403: Forbidden.
```

Attempting a yum install gives me this error - 
Command issued - 
`yum install http://download.zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm`

Error - 
```
Loaded plugins: fastestmirror, langpacks
Cannot open: http://download.zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm. Skipping.
Error: Nothing to do
```

Am I doing something wrong?

**Comment by rincebrain** [comment, marked as answer]
That's not [the URL](https://zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm) in [the documentation](https://openzfs.github.io/openzfs-docs/Getting%20Started/RHEL-based%20distro/index.html)? Where did you come upon that one?

**Reply by saumitrasapre** [reply]
I have been downloading ZFS using this URL until now 😅...

I encountered this issue recently.... 

Similarly, the [URL for EL7_8](https://zfsonlinux.org/epel/zfs-release.el7_8.noarch.rpm) is also different in the documentation, but this still works- 
`yum install http://download.zfsonlinux.org/epel/zfs-release.el7_8.noarch.rpm`

Wgetting the same url by changing the el7_X works for 7_8, 7_7, 7_6 , 7_5 too... Despite the URLs in the documentation being different...

I thought this would work for el7_9 too 😅

**Reply by rincebrain** [reply]
As I recall, download.zfsonlinux.org was the URL quite a long time ago indeed, and just points to S3. But as it's not the one referenced anywhere any more, I'm not surprised if it stopped getting new content.

**Reply by saumitrasapre** [reply]
Ohh... Okay... I'll use the new URL then...
Thank you so much for your reply!
